tools/twix/src/panels/look_at.rs

```diff
@@ -0,0 +1,220 @@
+                    if let Some(dimensions) = current_field_dimensions {
+                        let half_field_length = dimensions.length / 2.0;
+                        point![half_field_length, 0.0]
+                    } else {
+                        DEFAULT_TARGET
+                    }
+                }
+                LookAtType::Manual => {
+                    let max_dimension = current_field_dimensions
+                        .map_or(10.0, |dimensions: FieldDimensions| dimensions.length);
+
+                    ui.add(
+                        Slider::new(
+                            &mut self.look_at_target.x,
+                            RangeInclusive::new(-max_dimension, max_dimension),
+                        )
+                        .text("x")
+                        .smart_aim(false),
+                    );
+                    ui.add(
+                        Slider::new(
+                            &mut self.look_at_target.y,
+                            RangeInclusive::new(-max_dimension, max_dimension),
+                        )
+                        .text("y")
+                        .smart_aim(false),
+                    );
+
+                    self.look_at_target
+                }
+            };
+
+            ui.add_enabled_ui(self.is_enabled, |ui| {
+                if ui.button("Send Command").clicked() {
+                    send_standing_look_at(
+                        self.nao.as_ref(),
+                        self.look_at_target,
+                        self.camera_position,
+                    );
+                }
+            });
+
+            self.motion_command
+                .as_ref()
+                .map(|buffer| match buffer.get_latest() {
+                    Ok(value) => {
+                        let motion_command: MotionCommand = serde_json::from_value(value).unwrap();
+                        let contents = if let MotionCommand::SitDown {
+                            head: HeadMotion::LookAt { target, camera },
+                        }
+                        | MotionCommand::Stand {
+                            head: HeadMotion::LookAt { target, camera },
+                            ..
+                        }
+                        | MotionCommand::Walk {
+                            head: HeadMotion::LookAt { target, camera },
+                            ..
+                        }
+                        | MotionCommand::InWalkKick {
+                            head: HeadMotion::LookAt { target, camera },
+                            ..
+                        } = motion_command
+                        {
+                            format!(
+                                "Look at active: {{ target: {:?}, camera: {:?} }}",
+                                target, camera
+                            )
+                        } else {
+                            "Look at inactive".to_string()
+                        };
```

Comment:
Yup, that if expression also had me thinking for a bit, this was copied from Maxi's original implementation.

The status string just say what's the current motion command w.r.t. look at, but as you said, could only cover if the panel instructs this.

Reply:
I cleared this up, felt that printing this is not really necessary, but retained the status text label to show other messages.

Reply:
Thanks, looks a lot better now.
But I think it would be good to always show the current motion.
Maybe in a different color if it is currently being overridden.
That allows you to both see what motion the robot is performing when you claim its unsafe to override as well as seeing what actual values are being set by the panel.

Reply:
That sounds like a good idea, perhaps a summarised version.
But once you send the injected command, I think the motion command we recieve is the overridden one, maybe we can check if the basic structure is same as overriding or not and set the colours.